misc/testerina/modules/testerina-core/src/main/ballerina/execute.bal

```diff
@@ -293,13 +293,12 @@ function skipDataDrivenTest(TestFunction testFunction, string suffix, TestType t
     if (!suffixMatch) {
         string[] subTests = filterSubTests.get(functionKey);
         foreach string subFilter in subTests {
-
             string updatedSubFilter = subFilter;
             if (testType == DATA_DRIVEN_MAP_OF_TUPLE) {
                 if (subFilter.startsWith(SINGLE_QUOTE) && subFilter.endsWith(SINGLE_QUOTE)) {
                     updatedSubFilter = subFilter.substring(1, subFilter.length() - 1);
                 } else {
-                    continue;
+                    updatedSubFilter = subFilter;
```

Comment:
The variable `updateSubFilter` is initialised with `subFilter` we don't need to re-assign it again to the same value. We also can remove the else part.